bot/src/auctions.ts

```diff
@@ -24,7 +26,15 @@ export const getNewAuctionsFromActiveAuctions = function (activeActions: Auction
 };
 
 export const getAllAuctions = async function (network: string): Promise<AuctionInitialInfo[]> {
-    const auctions = await fetchAllInitialAuctions(network);
+    let collateralWhitelist: undefined | string[];
+    if (WHITELISTED_COLLATERALS) {
+        collateralWhitelist = parseCollateralWhitelist(WHITELISTED_COLLATERALS);
+        console.info(`auctions: whitelist is enabled, only fetching auctions of type "${WHITELISTED_COLLATERALS}"`);
```

Comment:
This console log should be inside setup, otherwise it will be printed all the time, right?

Reply:
I would actually reprint it every time auctions are fetched. If someone where to enter the logs of the bot, after it has already been running for a few days, it might be nice to easily see which auctions are being fetched in the console. If we only show it on setup the logs would be too far up. 

What do you think?

Reply:
I think it's ok to print it only once, but if you want, you can maybe add this info to the existing log about fetching? https://github.com/sidestream-tech/unified-auctions-ui/blob/2858a7e72d656e218ac2a0e497faaa82fe580196/bot/src/auctions.ts#L31

Reply:
I thought about this again and I actually changed my mind. I noticed that if the user has a very long whitelist, that this is going to spam the console way too much. 

Therefore I agree with you, that it does not make sense to display it with every fetch.